File: Python/URL/get_query_from_url.py
import urllib.parse


# Get the query from a given url.
def get_query_from_url(url):
    return urllib.parse.urlparse(url).query


def main():
    # Get the query from a given url.
    print(get_query_from_url(
        "https://www.example.com?username=user&password=pass"))


main()

File: TypeScript/URL/get-query-from-url.ts
import * as url from "url"

function main(): void {
    // Get the query from a given url.
    console.log(getQueryFromURL("https://www.example.com?query=true"))
}

main()

// Get the query from a given url.
function getQueryFromURL(givenURL: string): string | null {
    return new URL(givenURL).search
}
